src/coreComponents/fileIO/coupling/ChomboCoupler.cpp

```diff
@@ -57,35 +62,45 @@ void ChomboCoupler::write( double dt )
   }
 
   arrayView1d< integer const > const & ruptureState = faces->getReference< integer_array >( "ruptureState" );
-  arrayView1d< integer const > const & ghostRank = faces->getReference< integer_array >( faces->viewKeys.ghostRank );
+  arrayView1d< integer const > const & ghostRank = faces->ghostRank();
+
+  localIndex voidRegionIndex = -1;
+  elemRegionManager->forElementRegionsComplete( [&]( localIndex const elemRegionIndex,
+                                                     ElementRegionBase const & elemRegion )
+  {
+    if( elemRegion.getName() == "void" )
```

Comment:
Is this some sort of convention for input files using Chombo to have a region named "void"?